tiledb/sm/filter/test/tile_data_generator.h

```diff
@@ -43,6 +43,13 @@ using namespace tiledb::common;
 
 namespace tiledb::sm {
 
```

Comment:
This should be implemented in a test library and not in dev code.

Reply:
Done, I created whitebox_helpers.h to keep it in a header and avoid linking with the `tiledb_test_support_lib` library.